modules/hangman.js

```diff
@@ -64,12 +71,10 @@ class MtgHangman {
             'reactions to pick letters.';
 
         // instantiate embed object
-        const embed = new Discord.MessageEmbed({
-            author: {name: "Guess the card:"},
-            title,
-            description,
-            footer: {text: "You have "+this.gameTime/60000+" minutes to guess the card."}
-        });
+        const embed = new Discord.MessageEmbed()
+            .setTitle(title)
+            .addFields({name: 'Guess the card:', value: description})
+            .setFooter('You have ' + this.gameTime / 60000 + ' minutes to guess the card.');
```

Comment:
I'm not sure these changes are necessary. I updated the Discord library and the original code still works, plus moving the "Guess the card" title to the `name` field will push it *below* the card name, which makes it look weird when you've finished the game:
![image](https://user-images.githubusercontent.com/325521/79684172-9dc18280-822f-11ea-80ae-f0c1ebcca015.png)

Reply:
Are you certain you installed the new discord.js in your testing environment? If so, I'll have to look into why it broke for me and not you.

Reply:
Looking into my local `node_modules/discord.js/package.json` gives me this:
```  "_resolved": "https://registry.npmjs.org/discord.js/-/discord.js-12.2.0.tgz",
  "_shasum": "31018732e42a495c92655055192221eab2ad11a9",
  "_spec": "discord.js@^12.2.0",```
So I'm pretty sure that it's the most recent version. I also had to fix the server count due to the updated package.